linalg/densemat.cpp

```diff
@@ -2477,8 +2477,8 @@ void DenseMatrix::CopyMNt(const DenseMatrix &A, int row_offset, int col_offset)
    int i, j;
    double *v = A.data;
 
-   for (i = 0; i < A.Width(); i++)
-      for (j = 0; j < A.Height(); j++)
+   for (j = 0; j < A.Height(); j++)
```

Comment:
The rest are OK, but with this order this function is incorrect. Am I missing something?